app.go

```diff
@@ -814,8 +817,7 @@ func (app *App) ListenTLS(addr, certFile, keyFile string) error {
 			return fmt.Errorf("tls: cannot load TLS key pair from certFile=%q and keyFile=%q: %s", certFile, keyFile, err)
 		}
 		config := &tls.Config{
-			MinVersion:               tls.VersionTLS12,
-			PreferServerCipherSuites: true,
+			MinVersion: tls.VersionTLS13,
```

Comment:
would this be a breaking change?

Reply:
I'm not sure but it would be breaking change for old certficates may be.

Reply:
`MinVersion: tls.VersionTLS12`
`MaxVersion: tls.VersionTLS13`

can it prevent breaking change ?

Reply:
I think we dont need it. Also i've seen https://tip.golang.org/doc/go1.18#tls10. I think we can use tls1.2 as minimum version instead. I can revert it. @ReneWerner87 @hi019